composer.json

```diff
@@ -65,23 +62,27 @@
         "sonata-project/block-bundle": "^4.0",
         "sonata-project/doctrine-mongodb-admin-bundle": "^4.0",
         "sonata-project/doctrine-orm-admin-bundle": "^4.0",
+        "symfony/asset": "^4.4 || ^5.3 || ^6.0",
```

Comment:
Why do we need this dependency? We don't have any public resources here

Reply:
Because we are rendering templates of the admin bundle, that uses assets, and when adding framework bundle as a requirement is started failing, I had to add a manual config to enable assets explicitly here, so at the end this bundle requires assets for its test to execute.